gp-includes/things/translation.php

```diff
@@ -661,6 +661,23 @@ public function last_modified( $translation_set ) {
 		return $last_modified;
 	}
 
+	/**
+	 * Get IDs of translations by ID of their original and translation set.
+	 *
+	 * @since 2.3.0
+	 *
+	 * @param int $original_id        ID of original.
+	 * @param int $translation_set_id ID of translation set.
+	 * @return array $ids An array of IDs of matching translations.
+	 */
+	public function ids_by_original_and_set( $original_id, $translation_set_id  ) {
+		global $wpdb;
+
+		$ids = $wpdb->get_col( $wpdb->prepare( "SELECT id FROM {$this->table} WHERE original_id = %d AND translation_set_id = %d",  $original_id, $translation_set_id ) );
```

Comment:
Because I didn't see this as a final solution, just one example how to do. Also, at first I though about getting `status` column and then using that information (1 fuzzy, 2 waiting, 1 old).